src/components/course/data/utils.jsx

```diff
@@ -118,3 +111,15 @@ export function getProgramIcon(type) {
 }
 
 export const numberWithPrecision = (number, precision = 2) => number.toFixed(precision);
+
+// See https://openedx.atlassian.net/wiki/spaces/WS/pages/1045200922/Enroll+button+and+Course+Run+Selector+Logic
+// for more detailed documentation on course run selection and the enroll button.
+export function getActiveCourseRun(course) {
+  return course.courseRuns.find(courseRun => courseRun.uuid === course.advertisedCourseRunUuid);
+}
+
+export function getActiveCourseRuns(course) {
```

Comment:
Nit: should this be named something else?  Maybe `getAvailableCourseRuns(course)` to help distinguish it from `getActiveCourseRun`?

Reply:
I was trying to keep the variable naming here consistent with the B2C naming, but I do agree that it's confusing. Happy to change it if people find it helpful

Reply:
Went ahead and updated, I like the rename more